frontend/src/components/EditProfile/EditProfile.js

```diff
@@ -0,0 +1,147 @@
+import Axios from 'axios'
+import React, { useState, useEffect } from 'react'
+import { Redirect } from 'react-router-dom';
+import '../../config';
```

Comment:
In accordance to the coding standards, would it be possible to move `import '../../config';` to the top of the file and leave a line between those imports and the stylesheet import?